pkg/awsManager/vpcVpnManager.go

```diff
@@ -434,24 +408,22 @@ func DeleteNetworkAcl(client clientpkg.Client, vpcId *string) error {
 	return nil
 }
 
-func DeleteSecurityGroups(client clientpkg.Client, vpcId *string) error {
+func deleteSecurityGroups(client clientpkg.Client, vpcId *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file